src/lib/CLValue/PublicKey.ts

```diff
@@ -120,8 +125,14 @@ export class CLPublicKey extends CLValue {
     return this.tag === CLPublicKeyTag.SECP256K1;
   }
 
-  toHex(): string {
-    return `0${this.tag}${encodeBase16(this.data)}`;
+  toHex(checksummed = false): string {
```

Comment:
hm, I think it can be `true` by default, because if `fromHex` accepts also non-checksummed variant it shouldn't break anything for anybody. but we will add a note that next release will contain this change.